lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -13,7 +13,14 @@
 
 from os.path import expanduser
 
-from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+from ansible.module_utils.basic import AnsibleModule
```

Comment:
so where's basic.py?

Reply:
basic.py is in module_utils, but older versions don't have missing_required_lib definition

Reply:
perhaps we should have a condition "if ansible_version<2.8" instead of defining this stub function ....? what do you think?